modular/blocksyncer/modules/permission/permission_handler.go

```diff
@@ -39,6 +39,7 @@ var actionTypeMap = map[permissiontypes.ActionType]int{
 	permissiontypes.ACTION_UPDATE_GROUP_MEMBER: 9,
 	permissiontypes.ACTION_DELETE_GROUP:        10,
 	permissiontypes.ACTION_UPDATE_OBJECT_INFO:  11,
+	permissiontypes.ACTION_UPDATE_GROUP_EXTRA:  12,
```

Comment:
why name it update group extra? Can we change it to a name with more information of what it is doing?

Reply:
it is defined by chain, i don't have permission to change it
<img width="930" alt="image" src="https://github.com/bnb-chain/greenfield-storage-provider/assets/122767193/49bb1790-a82f-4285-9898-f776d9596817">